umbraco/Umbraco-CMS
# Add exclusion filter setting to typefinder settings

Fixes https://github.com/umbraco/Umbraco-CMS/issues/13440

# Notes

- Added exclusion filter setting to typefinder settings.
- Added new typefinder constructor.
- Obsoleted old typefinder constructor.
- Used exclusion filter in new typefinder constructor.

# How to test

- Add ABCPdf 12.4 Nuget package to your project.
- Add the following to your appsettings.json:
```
"Umbraco": {
    "CMS": {
      "TypeFinder": {
        "AssembliesAcceptingLoadExceptions": "*",
        "AdditionalAssemblyExclusionEntries": [ "WindowsBase", "PresentationFramework", "ReachFramework", "PresentationCore" ]
      },
...
```
- You should now be able to launch your Umbraco project.
